kolibri/utils/server.py

```diff
@@ -480,9 +480,12 @@ def get_urls(listen_port=None):
             __, __, port = get_status()
         urls = []
         if port:
-            interfaces = ifcfg.interfaces()
-            for interface in filter(lambda i: i["inet"], interfaces.values()):
-                urls.append("http://{}:{}/".format(interface["inet"], port))
+            try:
+                interfaces = ifcfg.interfaces()
+                for interface in filter(lambda i: i["inet"], interfaces.values()):
+                    urls.append("http://{}:{}/".format(interface["inet"], port))
+            except RuntimeError:
+                logger.error("Error retrieving network interface list!")
```

Comment:
Agreeing on the work-around! The upstream issue should be addressed, too, of course, but it's slightly complicated.